server/src/termdb.server.init.js

```diff
@@ -471,9 +471,28 @@ export function server_init_db_queries(ds) {
 		union all  
 		select term_id, value 
 		from anno_integer 
-		where sample=? ${termClause} ) join terms on terms.id = term_id`
+		where sample=? ${termClause}
+		union all 
+		select term_id, value 
```

Comment:
should also select `min_years_to_event` and display it on client

Reply:
Should be fixed now